src/DocumentDB.ChangeFeedProcessor/IUnhealthyHandlingStrategy.cs

```diff
@@ -0,0 +1,24 @@
+﻿//----------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.  Licensed under the MIT license.
+//----------------------------------------------------------------
+
+namespace Microsoft.Azure.Documents.ChangeFeedProcessor
```

Comment:
I think both new files and their namespaces should be under PartitionManagement, right?

Reply:
I moved it under Monitoring, wdyt?

Reply:
I think Monitoring should be OK.